src/IObjectMakr.ts

```diff
@@ -63,6 +63,13 @@ export interface IObjectMakrSettings {
      * be run when made.
      */
     onMake?: string;
+
+    /**
+     * Optionally, a custom functions mapping of constructors for the classes can be passed in.
+     * Functions passed in would not be redefined by processFunctions. This allows for pre-existing
+     * classes to be manually manipulated.
```

Comment:
So just changing to this:
'Optionally, existing classes can be passed in instead of using auto-generated ones.' ?